src/main/java/kr/hhplus/be/server/external/dto/naver/NaverSearchRequest.java

```diff
@@ -0,0 +1,40 @@
+package kr.hhplus.be.server.external.dto.naver;
+
+public class NaverSearchRequest {
+    private String query;
+    private String location;
+    private int display = 10;
+    private int start = 1;
+    private String sort = "random"; // random, comment
```

Comment:
```suggestion
    private SortType sort = SortType.RANDOM; // random, comment
```

타입을 enum 으로 관리하는건 어떨까요?